app/models/cohort_list.rb

```diff
@@ -22,24 +22,25 @@ class CohortList
     CHILD_NHS_NUMBER
   ].freeze
 
-  REGISTRATION_ATTRS = %i[
-    created_at
-    location_id
-    location_name
-    parent_name
-    formatted_parent_relationship
-    parent_email
-    parent_phone
-    first_name
-    last_name
-    common_name
-    date_of_birth
-    address_line_1
-    address_line_2
-    address_town
-    address_postcode
-    nhs_number
-  ].freeze
+  REGISTRATION_ATTRS =
+    [
+      :created_at,
+      :location_id,
+      :location_name,
+      :parent_name,
+      :formatted_parent_relationship,
+      :parent_email,
+      %i[parent_phone quoted],
+      :first_name,
+      :last_name,
+      :common_name,
+      :date_of_birth,
+      :address_line_1,
+      :address_line_2,
+      :address_town,
+      :address_postcode,
+      :nhs_number
+    ].map { _1.is_a?(Array) ? _1 : [_1, nil] }.freeze
```

Comment:
This bit may seem over-engineered, but I was mostly focused on keeping the list of attrs as clean as readable as possible so it's very clear which attrs are quoted. There are a couple of other ways to do this, but this seems cleanest.

Open to comments and suggestions to improve this.